### PR TITLE
improving x button

### PR DESCRIPTION
**Pull request by deadlybutter**
screen shot 2017-06-15 at 11 41 44 am
screen shot 2017-06-15 at 11 41 24 am

### CODE REVIEW

File: resources/assets/components/Modal/modal.scss [hunk 1]
```diff
@@ -30,11 +30,18 @@
 
   .modal__exit {
     position: absolute;
-    top: $base-spacing;
-    left: $base-spacing; // TODO: Move this to the right.
+    top: 2%;
```
Comment (weerd) [inline resources/assets/components/Modal/modal.scss]: Just curious but why `%` for the `top` and `right` and not a fixed value like `$half-spacing` or `$base-spacing`?

Comment (deadlybutter) [inline resources/assets/components/Modal/modal.scss]: @weerd originally i did percents to solve it looking weird on mobile, but i ended up adding that media query so i could go back to `$base-spacing`

**Comment by deadlybutter** [comment]
@weerd updated w/ static values, looks fine for me still!

File: resources/assets/components/Modal/modal.scss (hunk 1 above)
Comment (weerd) [inline resources/assets/components/Modal/modal.scss]: Yeah, I'd rather stay to a know value and be more consistent w/ the spacing across the page :)

**Comment by ngjo** [comment]
[🐛 Move the X into the modal 🐛](https://trello.com/c/kIonVAcS/697-%F0%9F%90%9B-move-the-x-into-the-modal-%F0%9F%90%9B)

**Pull request merged**